dicoogle/src/main/java/pt/ua/dicoogle/plugins/PluginController.java

```diff
@@ -487,6 +487,30 @@ public IndexerInterface getIndexerByName(String name, boolean onlyEnabled){
     	logger.error("No indexer matching name {} for onlyEnabled = {}", name, onlyEnabled);
     	return null;
     }
+
+    public JettyPluginInterface getServletByName(String name, boolean onlyEnabled){
+        Collection<JettyPluginInterface> plugins = getServletPlugins(onlyEnabled);
+        for(JettyPluginInterface p : plugins){
+            if(p.getName().equalsIgnoreCase(name)){
+                //logger.info("Retrived Query Provider: "+name);
+                return p;
+            }
+        }
+        logger.error("No indexer matching name {} for onlyEnabled = {}", name, onlyEnabled);

```

Comment:
This is not severe enough to be at ERROR level. I say we keep this (and the one below) at level INFO or DEBUG.

Reply:
I followed the behavior of the similar methods already implemented. Should I change the others too?

Reply:
That is odd. Since this can be triggered at the user's "fault", it is unreasonable to log these occurrences at the plugin controller and call them errors. Returning `null ` at the event of the plugin not existing is part of the contract. I say we use DEBUG for this entry and replace the others on a new PR (just in case other collaborators may wish to comment on this).